src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -576,7 +588,7 @@ def _job_settings(self, step_name: str, remote_wheel: str):
         version = version if not self._ws.config.is_gcp else version.replace("+", "-")
         return {
             "name": self._name(step_name),
-            "tags": {"version": f"v{version}"},
+            "tags": {"version": f"v{version}", **remove_after_tag},
```

Comment:
would it be cleaner to introduce `tags = {"version": f"v{version}"}` and expand it in a condition?